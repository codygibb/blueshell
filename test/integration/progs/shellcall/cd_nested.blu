$> mkdir -p .tmp/foo/bar/baz
$> mkdir -p .tmp/foo/meow/catz

func cwd() {
	$> basename $PWD
}

cd '.tmp/foo' {
	cwd()
	cd 'bar' {
		cwd()
	}
	cwd()
	cd 'meow' {
		cwd()
		cd 'catz' {
			cwd()
		}
		cwd()
	}
	cwd()
}
cwd()
