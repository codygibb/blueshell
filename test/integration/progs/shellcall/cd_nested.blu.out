foo
bar
foo
meow
catz
meow
foo
blueshell
